src/app/copmonents/header/header.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="header">
+  <div class=""><img src="../../../assets/img/sapex.png" alt=""></div>
+  <div class="candidates" *ngIf="role"><button mat-raised-button>Candidates list</button></div>
+  <div class="calendar" *ngIf="role"><button mat-raised-button>Calendar</button></div>
+  <div class="sandbox" *ngIf="role"><button mat-raised-button>New sandbox</button></div>
```

Comment:
Classes should describe element, in this case it's better to name it header-button. Also even better to use header html element, ul and li.